addons/website_slides/models/slide_channel.py

```diff
@@ -97,6 +97,7 @@ def _default_access_token(self):
     total_votes = fields.Integer('# Votes', compute='_compute_slides_statistics', store=True)
     total_time = fields.Float('# Hours', compute='_compute_slides_statistics', digits=(10, 4), store=True)
     # configuration
+    allow_rating = fields.Boolean("Allow Rating on Content", default=False, help="When activated, comments and review on slides of the channel will be allowed.")
```

Comment:
Rename to allow_comments as it will be used to trigger various stuff depending on the channel type